EventFilter/HcalRawToDigi/plugins/HcalRawToDigi.cc

```diff
@@ -37,6 +37,16 @@ HcalRawToDigi::HcalRawToDigi(edm::ParameterSet const& conf):
     fedUnpackList_.push_back(1118);
     fedUnpackList_.push_back(1120);
     fedUnpackList_.push_back(1122);
+    // HBHE uTCA
```

Comment:
Those explicitly listed are certainly what we need. 
Let me clarify it with HCAL OPS guys whether there could be some 
unnecessary "extra" FEDs still connected (presumably no)


On Thu, 18 Feb 2016, Slava Krutelyov wrote:

> 
> In EventFilter/HcalRawToDigi/plugins/HcalRawToDigi.cc:
> 
> > @@ -37,6 +37,16 @@ HcalRawToDigi::HcalRawToDigi(edm::ParameterSet const& conf):
> >      fedUnpackList_.push_back(1118);
> >      fedUnpackList_.push_back(1120);
> >      fedUnpackList_.push_back(1122);
> > +    // HBHE uTCA
> 
> can we instead of this change to a loop from MINHCALuTCAFEDID to MAXHCALuTCAFEDID ?
> that's from 1100 to 1199.
> The only reason not to would be that you'd expect the FEDs not listed here already to come and go and produce corrupt
> data.
> 
> —
> Reply to this email directly or view it on GitHub.[AEx02iVbeFXR0hFWg0luflN0Hz6WF_7Yks5plcrwgaJpZM4HdCHO.gif]
> 
> 
>